src/common/actions/snaps.js

```diff
@@ -9,16 +9,15 @@ export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
 export const FETCH_SNAPS_SUCCESS = 'FETCH_SNAPS_SUCCESS';
 export const FETCH_SNAPS_ERROR = 'FETCH_SNAPS_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps() {
```

Comment:
I've removed the `owner` parameter because this can be added to the request in the service layer. I have changed the Express handler for this API endpoint to add the owner from the session if it is not explicitly sent in the request.